src/palace/manager/api/authentication/basic.py

```diff
@@ -777,15 +824,19 @@ def enforce_library_identifier_restriction(
             # Restriction field is blank, so everything matches.
             return patrondata
 
-        patrondata, field = self.get_library_identifier_field_data(patrondata)
-        if self._restriction_matches(
-            field,
+        patrondata, field_value = self.get_library_identifier_field_data(patrondata)
+        isValid, reason = self._restriction_matches(
```

Comment:
Minor: `isValid` is very javascript of you 😄, should probably be `is_valid`.

Reply:
Hahaha. 💯

Reply:
Okay. Updated that! Can you tell where my mind is right now?